src/component/splits/mod.rs

```diff
@@ -498,40 +516,58 @@ impl Component {
         for column in &self.settings.columns {
             settings
                 .fields
-                .push(Field::new("Column Name".into(), column.name.clone().into()));
+                .push(Field::new(
+                    "Column Name".into(),
+                    "The name of the column. This is shown at the top of the list if the option to show column labels is enabled.".into(),
+                    column.name.clone().into(),
+                ));
```

Comment:
Same with this setting - I see it in LiveSplit One even though "Show Column Labels" is disabled. (Seems similar to https://github.com/LiveSplit/livesplit-core/issues/802)

Reply:
Yeah most components don't hide stuff properly. We'll track it with that issue.